components/Entity.h

```diff
@@ -85,5 +85,4 @@ class Entity{
 	std::vector<std::shared_ptr<Entity>>& getChildren();
 
 	SMath::Matrix4 model();
-};
-
+};
```

Comment:
Don't remove newline at end of files, and add them, pls, if you see one missing.
FLAG